packages/media/src/test/getResourceSize.js

```diff
@@ -18,7 +18,10 @@
  * Internal dependencies
  */
 import getResourceSize from '../getResourceSize';
-import { FULLBLEED_HEIGHT, PAGE_WIDTH } from '../../../../constants';
+import {
+  FULLBLEED_HEIGHT,
+  PAGE_WIDTH,
+} from '../../../../assets/src/edit-story/constants';
```

Comment:
I guess it would be good to get the units package merged first.

Reply:
I am hoping to merge unit package before this and rebase.